src/vm/wren_compiler.c

```diff
@@ -1096,13 +1096,16 @@ static void nextToken(Parser* parser)
       case '0':
         switch (peekChar(parser))
         {
-          case 'x': 
+          case 'x':
+          case 'X':
```

Comment:
I wouldn't allow both for the inconsistency it creates. Not just within user code, but because \u and \U have different meanings in wren already. we also have \x and that does not have a capital variant. I'd remove the uppercase.